src/bin/capsule.rs

```diff
@@ -60,6 +60,19 @@ fn group_contracts_by_type(contracts: Vec<Contract>) -> HashMap<TemplateType, Ve
     contracts_by_type
 }
 
+fn get_last_args() -> (Vec<String>, Option<Vec<String>>) {
+    let args: Vec<String> = env::args().collect();
+    let mut iter = args.splitn(2, |n| n == "--");
+    (iter.next().unwrap().to_vec(), {
+        let next_iter = iter.next();
+        if next_iter.is_none() || next_iter.unwrap().is_empty() {
+            Option::None
+        } else {
+            Option::Some(next_iter.unwrap().to_vec())
+        }
+    })
```

Comment:
```rust
(iter.next().unwrap().to_vec(), iter.next().map(|args| args.to_vec()))
```

Actually, the second return value can be `Vec<String>`, since an empty `Vec` may better than `None`, consider follow code:
``` rust
let build_cmd = [build_args_opt.to_string(), build_args].concat().join(" ");
```